custom_components/smartthinq_sensors/wideq/ac.py

```diff
@@ -882,11 +903,20 @@ def duct_zones_type(self):
             return None
         return self.to_int_or_none(self._data.get(DUCT_ZONE_V1_TYPE))
 
+    @property
+    def mode_jet(self):
+        key = self._get_state_key(STATE_MODE_JET)
+        if (value := self.to_int_or_none(self._data.get(key))) is None:
+            return None
+        status = value in (self.to_int_or_none(MODE_JET_COOL), self.to_int_or_none(MODE_JET_HEAT))
```

Comment:
Please do not convert constant. Eventually convert `value` in `str` that we know is `int` from prev method:
`str(value) in (MODE_JET_COOL, MODE_JET_HEAT)`